### ISSUE TITLE
No improvements of mAP during training 

### ISSUE THREAD

**Issue by algoteam5** [XiSHEN0220/SSR#6]
I was training the model with my dataset. As the logs say, there were no improvements of mAP during training. Why was that? Or was it because I didn't have the `test` module?
![image](https://user-images.githubusercontent.com/83445219/152269185-181dc0a7-a9da-47f0-9ee8-95f69be4b065.png)

**Comment by XiSHEN0220** [comment]
Hi, 

One could try with `a smaller learning rate` and/or `reducing a little bit of the weight decay` and/or `a small number of neighbors (e.g. 50)` and `the update step = 1` to see whether it can converge.  


It would be more efficient to help you with providing more training details. 
Also, SSR learns updates from neighbors, the neighbors need to contain numerous matched samples.

**Comment by algoteam5** [comment]
Thank you for your answer. I am training again with those changes. I will get back to you if I still face the issue.
